components/gitpod-db/src/typeorm/migration/1690915807191-AddGitStatusColumnToWorkspaceInstance.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * Copyright (c) 2023 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License.AGPL.txt in the project root for license information.
+ */
+
+import { MigrationInterface, QueryRunner } from "typeorm";
+import { columnExists } from "./helper/helper";
+
+export class AddGitStatusColumnToWorkspaceInstance1690915807191 implements MigrationInterface {
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        if (!(await columnExists(queryRunner, "d_b_workspace_instance", "gitStatus"))) {
+            await queryRunner.query("ALTER TABLE d_b_workspace_instance ADD COLUMN `gitStatus` text NULL");
```

Comment:
I think adding columns to big tables should always have `ALGORITHM=INPLACE, LOCK=NONE` in the ALTER clause. This table is highly used, and mysql even fails read operations when the table is recreated due to the altering.

Reply:
oh, good to know, should I add to the follow up PR? I copied it from other migrations, sorry

Reply:
All good, the migration happened on gitpod.io. I didn't see major issues.